contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -132,24 +93,29 @@ contract OptimismPortal {
         uint64 _gasLimit,
         bool _isCreation,
         bytes memory _data
-    ) public payable {
-        // Differentiate between sending to address(0)
-        // and creating a contract
-        if (_isCreation && _to != address(0)) {
-            revert NonZeroCreationTarget();
-        }
+    ) public payable metered(_gasLimit) {
+        // Just to be safe, make sure that people specify address(0) as the target when doing
+        // contract creations.
+        // TODO: Do we really need this? Prevents some user error, but adds gas.
+        require(
```

Comment:
It might be cheaper to do

```solidity
if (_isCreation) {
  require(_to == address(0));
}
```

See https://github.com/Rari-Capital/solmate/commit/d7f03abe94c51be3189014c8bdb89aa9bac14139#diff-5c8c7ec53b854a559eda8da5bde463ce7169b20e2f966fd4d721a4cd12e2c4b1R69

Reply:
Done